loading.bs

```diff
@@ -988,6 +1053,44 @@ A [=request=] |browserRequest| <dfn>matches the stored exchange</dfn>
     Behavior=] returning a list of lists.
 1. Return "match".
 
+<h3 algorithm id="queue-report">Queuing signed exchange report</h3>
+
+<dfn>Queuing signed exchange report</dfn> |report| is the following steps:
+
+1. Let |report body| and |policy| be the result of
+    [=generate a network error report=]</a> with |report|'s
+    [=signed exchange report/outer request=]. If the result is null, abort
```

Comment:
Ping? @dcreager

Reply:
I think that it should be that [fetch request](https://fetch.spec.whatwg.org/#concept-request) instance that is passed in.  We probably need to update NEL to just be defined in terms of fetch, instead of keeping the currently muddy language that tries to allow NEL to work with non-browser non-fetch HTTP clients.  (So for instance, to update NEL to have it pull out [current URL](https://fetch.spec.whatwg.org/#concept-request-current-url) to fill in the body of the report in a less hand-wavy way.)